src/Tizen.NUI/src/internal/WebView/WebBackForwardList.cs

```diff
@@ -50,23 +57,66 @@ public string Title
         {
             get
             {
-                return Interop.WebBackForwardListItem.GetTitle(SwigCPtr);
+                string ret = Interop.WebBackForwardListItem.GetTitle(SwigCPtr);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) return null;
+                return ret;
             }
         }
 
         /// <summary>
         /// Get original url.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public Uri OriginalUrl
+        public string OriginalUrl
         {
             get
             {
-                return new Uri(Interop.WebBackForwardListItem.GetOriginalUrl(SwigCPtr));
+                string ret = Interop.WebBackForwardListItem.GetOriginalUrl(SwigCPtr);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) return null;
+                return ret;
             }
         }
     }
 
+    /// <summary>
+    /// WebBackForwardCopiedListItems is a class for back-forward copied list item of web view.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class WebBackForwardCopiedItemList : Disposable
+    {
+        internal WebBackForwardCopiedItemList(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+        }
+
+        protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef swigCPtr)
+        {
+            Interop.WebBackForwardCopiedItemList.DeleteCopiedItems(swigCPtr);
+        }
+
+        /// <summary>
+        /// Get the count of copied items.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public uint GetCopiedItemsCount()
```

Comment:
Like `WebBackForwardList.ItemCount` could you make it a property?
```C#
public int ItemCount
```
And if you are going to provide it with `uint` type, please fix `WebBackForwardList.ItemCount` too.

Reply:
> Like `WebBackForwardList.ItemCount` could you make it a property?
> 
> ```cs
> public int ItemCount
> ```
> 
> And if you are going to provide it with `uint` type, please fix `WebBackForwardList.ItemCount` too.

Done